dotnetv3/Keyspaces/Tests/KeyspacesWrapperTests.cs

```diff
@@ -0,0 +1,164 @@
+                new ColumnDefinition { Name = "year", Type = "int" },
+                new ColumnDefinition { Name = "release_date", Type = "timestamp" },
+                new ColumnDefinition { Name = "plot", Type = "text" },
+            };
+
+            var partitionKeys = new List<PartitionKey>
+            {
+                new PartitionKey { Name = "year", },
+                new PartitionKey { Name = "title" },
+            };
+
+            _schemaDefinition = new SchemaDefinition
+            {
+                AllColumns = allColumns,
+                PartitionKeys = partitionKeys,
+            };
+        }
+
+        [Fact()]
+        [Order(1)]
+        public async Task CreateKeyspaceTest()
+        {
+            _keyspaceArn = await _wrapper.CreateKeyspace(_keyspaceName);
+
+            // Wait until keyspace has been created.
+            Thread.Sleep(10000);
+            var getKeyspaceArn = "";
+            do
+            {
+                getKeyspaceArn = await _wrapper.GetKeyspace(_keyspaceName);
+            } while (getKeyspaceArn != _keyspaceArn);
+            Assert.NotNull(_keyspaceArn);
+        }
+
+        [Fact()]
+        [Order(2)]
+        public async Task GetKeyspaceTest()
+        {
+            var keyspaceArn = await _wrapper.GetKeyspace(_keyspaceName);
+            Assert.Equal(keyspaceArn, _keyspaceArn);
+        }
+
+        [Fact()]
+        [Order(3)]
+        public async Task CreateTableTest()
+        {
+            var tableArn = await _wrapper.CreateTable(_keyspaceName, _schemaDefinition, _tableName);
+
+            // Wait for the table to be created.
+            Thread.Sleep(10000);
+            try
+            {
+                GetTableResponse resp;
+
+                // Wait for the table to be created.
+                do
+                {
+                    resp = await _wrapper.GetTable(_keyspaceName, _tableName);
+                } while (resp.Status != TableStatus.ACTIVE);
+            }
+            catch (ResourceNotFoundException ex)
+            {
+                Console.WriteLine($"Error: {ex.Message}");
+            }
+            Assert.NotNull(tableArn);
+        }
+
+        [Fact()]
+        [Order(4)]
+        public async Task GetTableTest()
+        {
+            var response = await _wrapper.GetTable(_keyspaceName, _tableName);
+            Assert.NotNull(response);
+        }
+
+        [Fact()]
+        [Order(5)]
+        public async Task UpdateTableTest()
+        {
+            _timeChanged = DateTime.UtcNow;
+            var resourceArn = await _wrapper.UpdateTable(_keyspaceName, _tableName);
+            Assert.NotNull(resourceArn);
+        }
+
+        [Fact()]
+        [Order(6)]
+        public async Task RestoreTableTest()
+        {
+            var resourceArn = await _wrapper.RestoreTable(_keyspaceName, _tableName, _timeChanged);
+            Assert.NotNull(resourceArn);
+        }
+
+        [Fact()]
+        [Order(49)]
```

Comment:
The test order goes from 6 to 49?

Reply:
I was leaving room for other tests that had to go in the middle. I've renumbered the tests.